README.md

```diff
@@ -2,7 +2,7 @@
 
 
 <div align="center">
-  <img src="my_own_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="" alt="logo" width="140"  height="auto" />
```

Comment:
- [OPTIONAL] Kindly consider removing it as it will make your readme file cleaner.